### PR TITLE
Fix args.all handling

### PR DESCRIPTION
**Pull request by andre-rosa**
- distro could never be 'all' after all

### CODE REVIEW

**Comment by andre-rosa** [comment]
@allenh1 ptal

File: tests/test_utils.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def test_trim_string(self):
         self.assertEqual(ret, 'a[...]')
 
     def test_mkdir(self):
-        """Tests the make directory funciton"""
+        """Tests the make directory function"""
```
Comment (allenh1) [inline tests/test_utils.py]: I prefer the spelling fixes be handled in their own PR if possible

File: superflore/generators/bitbake/run.py [hunk 2]
```diff
@@ -188,7 +189,7 @@ def main():
         delta = gen_delta_msg(total_changes)
         missing_deps = gen_missing_deps_msg(total_broken)
         # Commit changes and file pull request
-        overlay.commit_changes(args.ros_distro)
+        overlay.commit_changes('all' if args.all else args.ros_distro)
```
Comment (allenh1) [inline superflore/generators/bitbake/run.py]: I think this is the only change in this PR that is fixing an issue? If you could please file an issue for that problem as well that would be really great (for tracking/changelog generation for the next release).

**Comment by andre-rosa** [comment]
> Seems there's a lot of extra stuff in this PR -- If it does not depend on #180, I would strongly prefer you reduce it to only the issue it is addressing so we can get it out of the way of other contributions.
> 
> Thanks for the fix!

Thanks, yes; it is just rebased on top of other PRs. ;)

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,11 +30,11 @@
 **Implemented enhancements:**
 
 - Move some packages to DEPEND [\#83](https://github.com/ros-infrastructure/superflore/issues/83)
-- Create a `tar-archive-dir` to clean up the `.tar` files downloaded for Open Embedded [\#80](https://github.com/ros-infrastructure/superflore/issues/80)
```
[the remark this answers is not in the document]
Comment (andre-rosa) [reply CHANGELOG.md]: Updated in #181.

File: superflore/generators/bitbake/run.py (hunk 2 above)
Comment (andre-rosa) [reply superflore/generators/bitbake/run.py]: Yes, thanks! Fixes #183

**Comment by allenh1** [comment]
I'll check this one & then merge it

**Comment by allenh1** [comment]
Filed ros/ros-overlay#796 with this in place -- worked like a charm! (there were not any updates, so I just re-generated dashing).

Thanks for all the patches! We're getting closer!

**Comment by andre-rosa** [comment]
> Filed [ros/ros-overlay#796](https://github.com/ros/ros-overlay/pull/796) with this in place -- worked like a charm! (there were not any updates, so I just re-generated dashing).
> 
> Thanks for all the patches! We're getting closer!

Thanks a bunch, @allenh1!
